apps/frontend/src/index.tsx

```diff
@@ -38,7 +41,10 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							{
 								element: (
 									<ProtectedRoute
-										routePermissions={[PermissionKey.MANAGE_USER_ACCESS]}
+										routePermissions={[
+											PermissionKey.MANAGE_USER_ACCESS,
+											PermissionKey.MANAGE_ALL_PROJECTS,
+										]}
 									>
 										<AccessManagement />
```

Comment:
Do we need this permission for access management?

Reply:
From what i understood the person with Manage all Projects should be able to access it, but not the sidebar for access management, i believe in the future the managing of Project groups will be handled there, we do not have anything of it in project specification, so i left it like that, until further changes is implemented, perhaps it is better to remove it until start working on project groups?

Reply:
user with MANAGE_ALL_PROJECTS permission, won't be able to access root AccessManagement page, so we need to remove this

Reply:
This has now been adjusted